test/bin/.keep

```diff
@@ -0,0 +1 @@
+Do not remove. Ensures existence of directory.
```

Comment:
Curious why we need the directory?

Reply:
The `mysql_server_vault` CI job downloads the `vault` binary onto this directory. That CI fails when the directory does not exist.